pkgs/servers/mail/cyrus-imapd/default.nix

```diff
@@ -0,0 +1,22 @@
+{ stdenv, fetchurl, pkgconfig, perl, bison, flex, openssl, jansson, cyrus_sasl, icu }:
+
+stdenv.mkDerivation rec {
+  pname = "cyrus-imapd";
+  version = "3.0.10";
+
+  src = fetchurl {
+    url = "https://www.cyrusimap.org/releases/${pname}-${version}.tar.gz";
+    sha256 = "02q2j4qkj58gz4kjs0s19j6p1d6nhdh5pzrhkilgigny7pglpcm2";
+  };
+
+  nativeBuildInputs = [ pkgconfig perl ];
+  buildInputs = [ bison flex openssl jansson cyrus_sasl icu ];
```

Comment:
I haven't looked at the code for this project, but usually `bison` and `flex` are `nativeBuildInputs`. That isn't the case here?

Reply:
You're right, thanks ^^